assets/javascript/app.js

```diff
@@ -0,0 +1,199 @@
+      q4: 'What is the capital of the United Kingdom?',
+      q5: 'Tehran is the capital of which country?',
+      q6: 'What is the capital of China?',
+      q7: 'Santiago is the capital of which country?',
+      q8: 'What is the capital of Argentina?',
+      q9: 'Zagreb is the capital of which country?',
+      q10: 'What is te capital of South Sudan?',
+    },
+    options: {
+      q1: ['Georgia', 'Haiti', 'Iceland', 'Egypt'],
+      q2: ['Calcutta', 'Bombay', 'New Delhi', 'Madras'],
+      q3: ['Georgia', 'Gabon', 'Guatemala', 'Indonesia'],
+      q4: ['Birmingham', 'Glasgow', 'London', 'Oxford'],
+      q5: ['Iraq','Iran','Kenya','Kosovo'],
+      q6: ['Shanghai', 'Tokyo', 'Hong Kong', 'Beijing'],
+      q7: ['Congo', 'Croatia', 'Cuba', 'Chile'],
+      q8: ['Buenos Aires','La Paza','Montevideo', 'Santiago'],
+      q9: ['Croatia', 'Costa Rica', 'Estonia', 'Fiji'],
+      q10: ['Mogadishu', 'Khartoum', 'Juba', 'Asmara'],
+    },
+    answers: {
+      q1: 'Egypt',
+      q2: 'New Delhi',
+      q3: 'Georgia',
+      q4: 'London',
+      q5: 'Iran',
+      q6: 'Beijing',
+      q7: 'Chile',
+      q8: 'Buenos Aires',
+      q9: 'Croatia',
+      q10: 'Juba',
+      
+    },
+    // trivia methods
+    // method to initialize game
+    startGame: function(){
+      // restarting game results
+      trivia.currentSet = 0;
+      trivia.correct = 0;
+      trivia.incorrect = 0;
+      trivia.unanswered = 0;
+      clearInterval(trivia.timerId);
+      
+      // show game section
+      $('#game').show();
+      
+      //  empty last results
+      $('#results').html('');
+      
+      // show timer
+      $('#timer').text(trivia.timer);
+      
+      // remove start button
+      $('#start').hide();
+  
+      $('#remaining-time').show();
+      
+      // ask first question
+      trivia.nextQuestion();
+      
+    },
+    // method to loop through and display questions and options 
+    nextQuestion : function(){
+      
+      // set timer to 20 seconds each question
+      trivia.timer = 20;
+       $('#timer').removeClass('last-seconds');
+      $('#timer').text(trivia.timer);
+      
+      // to prevent timer speed up
+      if(!trivia.timerOn){
+        trivia.timerId = setInterval(trivia.timerRunning, 1000);
+      }
+      
+      // gets all the questions then indexes the current questions
+      var questionContent = Object.values(trivia.questions)[trivia.currentSet];
+      $('#question').text(questionContent);
+      
+      // an array of all the user options for the current question
+      var questionOptions = Object.values(trivia.options)[trivia.currentSet];
+      
+      // creates all the trivia guess options in the html
+      $.each(questionOptions, function(index, key){
+        $('#options').append($('<button class="option btn btn-info btn-lg">'+key+'</button>'));
+      })
+      
+    },
+    // method to decrement counter and count unanswered if timer runs out
+    timerRunning : function(){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method 'timerRunning' has a complexity of 6. (complexity)](https://app.codacy.com/app/columbia-fsf-f2019/trivia-game/pullRequest?prid=4555493)